src/components/MapArea.js

```diff
@@ -36,35 +36,22 @@ const StyleBar = styled.div`
   margin: 12px;
   border-radius: 4px;
 `
-// The GeoJSON data for the trail
-// This data is mocked
-// In the future make a call to an API to retrieve this data
-const trailData = {
-  "type": "Feature",
-  "geometry": {
-    "type": "MultiLineString",
-    "coordinates": [[[-75.51277, 45.39466, 75.0], [-75.51282, 45.39461, 75.0], [-75.51269, 45.39445, 74.0], [-75.51181, 45.39327, 73.0], [-75.51106, 45.3933, 75.0], [-75.51054, 45.39337, 74.0], [-75.51019, 45.39337, 73.0], [-75.50947, 45.39339, 73.0], [-75.50919, 45.3934, 74.0], [-75.50913, 45.39333, 73.0], [-75.5091, 45.39328, 73.0], [-75.50914, 45.39297, 69.0], [-75.50909, 45.39274, 69.0], [-75.50912, 45.39125, 69.0], [-75.50932, 45.39076, 69.0], [-75.51048, 45.39042, 67.0], [-75.51101, 45.39038, 67.0], [-75.51207, 45.39006, 68.0], [-75.51227, 45.39047, 69.0], [-75.51231, 45.39064, 69.0], [-75.51253, 45.39088, 68.0], [-75.5126, 45.39095, 68.0], [-75.51271, 45.39118, 67.0], [-75.51291, 45.39142, 68.0], [-75.5132, 45.39163, 68.0], [-75.51321, 45.39236, 66.0], [-75.51324, 45.39274, 70.0], [-75.51233, 45.39323, 72.0], [-75.51223, 45.39341, 73.0], [-75.51218, 45.39346, 73.0], [-75.51216, 45.39366, 74.0], [-75.51246, 45.39416, 73.0], [-75.51277, 45.39466, 75.0]]],
-    "bbox": [-75.50909, 45.39466, -75.51324, 45.39006]
-  },
-  "properties": {
-    "name": "Trail Planner Map",
-    "desc": "Generated by AllTrails https://www.alltrails.com/"
-  }
-};
 
-
-export default function MapArea() {
+export default function MapArea(props) {
   const mapContainer = useRef(null);
   const map = useRef(null);
-
-  const displayDataOnMap = (map, trailData) => {
+  const { selectedTrail } = props;
+    
+  const displayDataOnMap = (map, trail) => {
+    // Grab the trail data from the last element of trail.features
+    const trailData = trail.features[trail.features.length - 1];
```

Comment:
In this case we know that the last feature is the LineString / trail data. What if we didn't know the index of the item we wanted? This should be updated so that it is not dependent on that being the last item.